server/storage/mvcc/kv_view.go

```diff
@@ -21,6 +21,24 @@ import (
 	"go.etcd.io/etcd/server/v3/lease"
 )
 
+type readViewRTxMode struct {
+	KV
+	rTxMode ReadTxMode
+}
+
+func ReadViewRTxMode(rv KV, rTxMode ReadTxMode) *readViewRTxMode {
+	return &readViewRTxMode{
+		KV:      rv,
+		rTxMode: rTxMode,
+	}
+}
+
+func (rv *readViewRTxMode) Rev() int64 {
+	tr := rv.KV.Read(rv.rTxMode, traceutil.TODO())
+	defer tr.End()
+	return tr.Rev()
+}
+
```

Comment:
We don't need this change after using `SharedBufReadTxMode` for both `FirstRev` and `Rev`.